src/ocamlorg_web/lib/redirection.ml

```diff
@@ -449,6 +449,25 @@ let from_v2 =
     ("/releases/latest/api/index.html", Url.api_with_version latest_version);
   ]
 
+let default_index_html =
+  [
+    ("/manual", Url.manual_lang latest_version);
```

Comment:
We will need to add one for `latest`.

Reply:
We don't need a redirect for `latest`, we need to serve this via Dream under `/manual/latest`. A redirect would mean that the URL in the browser would change to the specific version.